documentation/configuring_bolt.md

```diff
@@ -1,59 +1,190 @@
+behaves, such as how many threads it can use when running commands on targets,
+and configures different components of the project, such as a list of plans
+and tasks that are visible to the user. Any directory containing a
+`bolt-project.yaml` file is automatically considered a [Project
+directory](bolt_project_directories.md).
 
-- **User**
+Project configuration files take precedence over `bolt.yaml` files. If a
+project directory contains both files, Bolt will only load and read
+configuration from `bolt-project.yaml`.
 
-  The `bolt.yaml` file at the user level applies settings only to that user.
-  Matching settings at the user level are overridden by project-level settings,
-  but take precedence over system-wide settings.  
+The `bolt-project.yaml` file supports most of the options that a `bolt.yaml`
+file does, with the exception of the following inventory configuration options:
 
-  `~/.puppetlabs/etc/bolt/bolt.yaml`
+- `transport`
+- `docker`
+- `local`
+- `pcp`
+- `remote`
+- `ssh`
+- `winrm`
 
-- **System-wide**
+You can view a full list of the available options in [Bolt configuration
+options](bolt_configuration_reference.md).
 
-  Settings in a system-wide config file apply to all users running Bolt,
-  regardless of the Bolt project directory. However, matching settings at the
-  project or user level override system-wide settings.
+### `inventory.yaml`
 
-  The `bolt.yaml` file at the system-wide level applies settings only to that
-  user.
+**Filepath:** `<project-directory>/inventory.yaml`
 
-  \*nix: `/etc/puppetlabs/bolt/bolt.yaml`
+The inventory file is a structured data file that contains groups of targets
+that you can run Bolt commands on, as well as configuration for the transports
+used to connect to the targets. Most projects will include an inventory file.
 
-  Windows: `%PROGRAMDATA%\PuppetLabs\bolt\etc\bolt.yaml`
+Inventory configuration can be set at multiple levels in an inventory file
+under a `config` option. The following options can be set under this option:
+
+- `transport`
+- `docker`
+- `local`
+- `pcp`
+- `remote`
+- `ssh`
+- `winrm`
+
+You can read more about inventory files and the available options in
+[Inventory files](inventory_file_v2.md).
+
+### `bolt.yaml`
+
+> **Note** The project-level `bolt.yaml` file is on the path towards
+> deprecation and will be removed in a future version of Bolt. Use
+> `bolt-project.yaml` and `inventory.yaml` files instead.
+
+**Filepath:** `<project-directory>/bolt.yaml`
+
+The Bolt configuration file can be used to set all available configuration
+options, including default inventory configuration options. Any directory
+containing a `bolt.yaml` file is automatically considered a [Project
+directory](bolt_project_directories.md).
+
+You can view a full list of the available options in [Bolt configuration
+options](bolt_configuration_reference.md).
+
+## User-level configuration
+
+Most configurable options in Bolt can be set at the user level. Configuration
```

Comment:
```suggestion
Configuration
```

Reply:
That line feels more confusing than illuminating, and raises questions like "What is user level configuration" and "Wait, but what options aren't included"